### PR TITLE
initialize custom Airbyte source connector for JazzHR [DE-305]

### PR DESCRIPTION
**Pull request by emily-flambe**
cc BANDDEs: @kanelouise @jitoquinto @royconst 

Looms??? LOOMS

[intro to project structure](https://www.loom.com/share/17e2f3704ab84653a85340d8f10e7142?sid=62cb12f0-ff0c-4ce4-a093-ee72f25dda96)

[CODE WALKTHROUGH](https://www.loom.com/share/67d7b9b4fba44c94b7787b5f71de5f15?sid=40d7b14c-f35f-4c63-98d4-82c78e389f22)

[testing locally](https://www.loom.com/share/ceafad804e91401e90803f1acfdb8643?sid=bdac8b36-7e4a-443d-81e2-416d58c657dd)

[building, pushing, and running docker images](https://www.loom.com/share/72cb5492fe4747869221682d902c1861?sid=9c25f953-ace5-4624-9ecf-6c296e468851)

### CODE REVIEW

**Review by kanelouise — APPROVED** [review]
Thanks for the loomies. I like that building these custom connectors gives us a peak under the hood at what AB is actually doing.

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+source-jazzhr = {editable = true, file = "file:///Users/emilycogsdill/Documents/GitHub/airbyte-source-jazzhr"}
```
Comment (jitoquinto) [inline Pipfile:7]: Should this path be removed?

**Review by jitoquinto — APPROVED** [review]
Thanks for making those looms! and lgtm!

File: Pipfile (hunk 1 above)
Comment (emily-flambe) [reply Pipfile:7]: oh, good catch!! it's not even clear what this was doing lol, removing now.

**Pull request merged**